src/commands/force/org/display.ts

```diff
@@ -28,7 +28,10 @@ export class OrgDisplayCommand extends SfdxCommand {
 
   public async run(): Promise<OrgDisplayReturn> {
     // translate to alias if necessary
-    const username = (await Aliases.fetch(this.flags.targetusername)) ?? (this.flags.targetusername as string);
+    const username =
+      (await Aliases.fetch(this.flags.targetusername)) ??
+      (this.flags.targetusername as string) ??
+      this.org.getUsername();
```

Comment:
Why don't you just always use `this.org.getUsername()`? It should do all the above for you.